app/Http/Controllers/WelcomeController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Contact;
+use App\Models\User;
+use App\Models\Level;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Storage;
+
+class WelcomeController extends Controller
+{
+    private $contact;
+    private $user;
+    private $level;
+
+    public function __construct(Contact $contact, User $user, Level $level){
```

Comment:
Please remove this since it's not being used and all other private declaration above